src/FastBoard.cpp

```diff
@@ -418,6 +450,9 @@ int FastBoard::get_stone_count() {
     return m_totalstones[BLACK] + m_totalstones[WHITE];
 }
 
+/*
+    @return estimate of the Monte-Carlo play-out score 
```

Comment:
That's what the function name says.

Reply:
OK. I removed the comment, but it was not automatically obvious to me when I first read the code what _mc_ stood for.